e2e_tests/test_browse_transferring_body.py

```diff
@@ -130,16 +142,20 @@ def test_browse_transferring_body_sort_functionality_by_series_descending(
         header_rows = standard_user_page.locator(
             "#tbl_result tr:visible"
         ).evaluate_all(
-            """els => els.slice(0).map(el =>
-              [...el.querySelectorAll('th')].map(e => e.textContent.trim())
-            )"""
+            """els => els.map(el => {
+                var headerCells = [...el.querySelectorAll('th')];
+                if (headerCells.length >= 3) {
+                    headerCells.splice(2, 1);
```

Comment:
why headerCelss.shift() has been used, can you not use the class to identify a row similar to the implementation in e2e_tests/test_search.py

rows = aau_user_page.locator("#tbl_result tr.top-row").evaluate_all(
            """els => els.slice(0).map(el =>
              [...el.querySelectorAll('td.search__mobile-table__top-row')].map(e => e.textContent.trim())
            )"""
        )

Reply:
resolved on call